api/lib/infrastructure/repositories/certification-repository.js

```diff
@@ -33,7 +33,7 @@ async function _getLatestAssessmentResult(certificationCourseId) {
       qb.where('assessments.certificationCourseId', '=', certificationCourseId);
     })
     .orderBy('createdAt', 'desc')
-    .fetch({ require: true });
+    .fetch({ require: false });
```

Comment:
On avait `require: true` ici, donc suivant la logique:

```suggestion
    .fetch();
```

Reply:
Ouh bien vu 🦅 @celineung , je corrige avant de merge